packages/builder/src/components/backend/DatasourceNavigator/DatasourceNavigator.svelte

```diff
@@ -152,10 +163,19 @@
 
     // Check for a matching view
     const selectedView = views.selected?.name
-    const table = options.find(table => {
+    const viewTable = options.find(table => {
       return table.views?.[selectedView] != null
     })
-    return table != null
+    if (viewTable) {
+      return true
+    }
+
+    // Check for a matching viewV2
+    const selectedViewV2 = viewsV2.selected?.name
+    const viewV2Table = options.find(table => {
+      return table.views?.[selectedViewV2] != null
```

Comment:
I don't think it will ever be null no, but the loosey `!= null` will account for both null and undefined. But you're right that an explicit `=== undefined` would also be fine.

I think this does need changed though because this would false positive if you had the same view name in 2 different tables. We should use the view's `tableId` property instead.